docs/cli.md

```diff
@@ -515,7 +515,7 @@ dependencies
  - pytzdata >=2017.2.2
 
 required by
- - calendar >=1.4.0
+ - calendar requires pendulum >=1.4.0
```

Comment:
```suggestion
 - calendar requires >=1.4.0
```

Should we omit the package name? (It will be the same in each line if it is required by multiple packages.)

@Secrus What is your opinion?

Reply:
Yeah, I think it's fine without the package name. Maybe we can add some explanation for this output in the docs too?

Reply:
OK, I've reverted the addition of the package name, so the PR now just adds the word 'requires' for each item in `Required by`.

I am not sure what I would add to docs for this change - there's already amended sample output, and now the output is more self-explanatory.

(I do think `show` documentation can be improved overall and I am drafting PR #9752 on that - I think I'm going to reduce scope though)